app/views/events/_guidelines.html.erb

```diff
@@ -0,0 +1,31 @@
+<h2 class="mb-10-only">Guidelines for *Bridge workshops</h2>
+  <p>
+  <a href="http://bridgefoundry.org/">Bridge Foundry</a>'s mission is to
+  empower people with technology through teaching and facilitating access,
+  enlarging the community of people who give back and teach others.
+  One of the ways we do that is by providing this online registration 
```

Comment:
'online registration system' sounds cold and is redundant with the next paragraph, maybe remove it?